src/all/batoto/src/eu/kanade/tachiyomi/extension/all/batoto/BatoTo.kt

```diff
@@ -309,23 +322,36 @@ open class BatoTo(
         }
         return super.mangaDetailsRequest(manga)
     }
+    private var titleRegex: Regex =
+        Regex("(?:\\([^()]*\\)|\\{[^{}]*\\}|\\[(?:(?!]).)*]|«[^»]*»|〘[^〙]*〙|「[^」]*」|『[^』]*』|≪[^≫]*≫|﹛[^﹜]*﹜|𖤍.+?𖤍|/.+?)\\s*|([|/~].*)", RegexOption.IGNORE_CASE)
 
     override fun mangaDetailsParse(document: Document): SManga {
         val infoElement = document.select("div#mainer div.container-fluid")
         val manga = SManga.create()
         val workStatus = infoElement.select("div.attr-item:contains(original work) span").text()
         val uploadStatus = infoElement.select("div.attr-item:contains(upload status) span").text()
-        manga.title = infoElement.select("h3").text().removeEntities()
+        val originalTitle = infoElement.select("h3").text().removeEntities()
+        val alternativeTitles = document.select("div.pb-2.alias-set.line-b-f").text()
+        val description = infoElement.select("div.limit-html").text() + "\n" +
+            infoElement.select(".episode-list > .alert-warning").text().trim()
+        val cleanedTitle = if (isRemoveTitleVersion()) {
+            originalTitle.replace(titleRegex) { matchResult ->
+                matchResult.groupValues.getOrNull(1)?.trim() ?: ""
+            }
```

Comment:
should it simply replace by empty string, I don't know what the lambda is trying to do here?

Reply:
Not sure why I did that as well tbh lmao
I'll fix this